instrumentation/jetty-httpclient/jetty-httpclient-12.0/testing/src/main/java/io/opentelemetry/instrumentation/jetty/httpclient/v12_0/AbstractJettyClient12Test.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jetty.httpclient.v12_0;
+
+import static java.util.logging.Level.FINE;
+
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.http.AbstractHttpClientTest;
+import io.opentelemetry.instrumentation.testing.junit.http.HttpClientInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.http.HttpClientResult;
+import io.opentelemetry.instrumentation.testing.junit.http.HttpClientTestOptions;
+import java.net.URI;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.logging.Logger;
+import org.eclipse.jetty.client.ContentResponse;
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.client.Request;
+import org.eclipse.jetty.client.Response;
+import org.eclipse.jetty.http.HttpField;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public abstract class AbstractJettyClient12Test extends AbstractHttpClientTest<Request> {
+
+  private static final Logger logger = Logger.getLogger(AbstractJettyClient12Test.class.getName());
+
+  @RegisterExtension
+  static final InstrumentationExtension testing = HttpClientInstrumentationExtension.forAgent();
+
+  protected abstract HttpClient createStandardClient();
+
+  protected abstract HttpClient createHttpsClient(SslContextFactory.Client sslContextFactory);
+
+  protected HttpClient client = createStandardClient();
+
+  protected HttpClient httpsClient;
+
+  Request jettyRequest = null;
+
+  @Override
+  protected void configure(HttpClientTestOptions.Builder optionsBuilder) {
+    try {
+      // disable redirect tests
+      optionsBuilder.disableTestRedirects();
+      // jetty 12 does not support to reuse request
+      // use request.send() twice will block the program infinitely
+      optionsBuilder.disableTestReusedRequest();
+      // start the main Jetty HttpClient and a https client
+      client.setConnectTimeout(CONNECTION_TIMEOUT.toMillis());
+      client.start();
+
+      SslContextFactory.Client tlsCtx = new SslContextFactory.Client();
+      httpsClient = createHttpsClient(tlsCtx);
+      httpsClient.setFollowRedirects(false);
+      httpsClient.start();
+    } catch (Throwable t) {
+      logger.log(FINE, t.getMessage(), t);
```

Comment:
It had better to adjust the log level from `log` to `error` here.

Reply:
The log has been deleted